libraries/WDT/examples/Example3_WDT_LowPower/Example3_WDT_LowPower.ino

```diff
@@ -0,0 +1,168 @@
+/*
+  Author: Adam Garbo
+  Created: August 1st, 2020
+  License: MIT. See SparkFun Arduino Apollo3 Project for more information
+
+  This example demonstrates the combined use of the Artemis Watchdog Timer (WDT)
+  and real-time clock (RTC).
+
+  Both RTC and WDT interrupts will wake the system, print the date and time,
+  and then re-enter deep sleep.
+
+  The WDT is configured to trigger every 10 seconds. If the WDT is not "pet" 
+  after 100 seconds, a system reset will be triggered.
+  
+  The RTC alarm is configured to trigger every minute and enter deep sleep 
+  between interrupts. Alarm interuptswill also restart the WDT and reset
+  the watchdog interrput counter.
+*/
+
+#include <RTC.h>
+#include <WDT.h>
+
+APM3_RTC rtc;
+APM3_WDT wdt;
+
+volatile bool alarmFlag = false; // RTC ISR flag
+volatile bool watchdogFlag = false; // Watchdog Timer ISR flag
+volatile int watchdogInterrupt = 0; // Watchdog interrupt counter
+
+void setup()
+{
+  Serial.begin(115200);
+
+  Serial.println("Artemis Watchdog Low Power Example");
+
+  // Set the RTC time using UNIX Epoch time
+  rtc.setEpoch(1596240000); // Saturday, August 1, 2020 00:00:00
+
+  // Set the RTC's alarm
+  rtc.setAlarm(0, 0, 0, 0, 0, 0); // Set alarm (hund, ss, mm, hh, dd, mm)
+  rtc.setAlarmMode(6); // Set the RTC alarm to trigger every minute
+  rtc.attachInterrupt(); // Attach RTC alarm interrupt
+
+  // Configure the watchdog timer
+  // See Example2_WDT_Config for more information on how to configure the watchdog
+  wdt.configure(2, 160, 240); // 16 Hz clock, 10-second interrupt period, 15-second reset period
```

Comment:
Use macro AM_HAL_WDT_LFRC_CLK_16HZ instead of 2?

Reply:
It's important to note that the `AM_HAL_WDT_LFRC_CLK_16HZ` definition cannot be used as a value to set the specific WDT CLKSEL register. These HAL definitions call `_VAL2FLD(WDT_CFG_CLKSEL, WDT_CFG_CLKSEL_128HZ)`, which sets a specific bit of a 32-bit variable that is later passed to the entire WDT CFG register.

For example:
```
printBinary(AM_HAL_WDT_LFRC_CLK_128HZ);
-> 00000001000000000000000000000000 

printBinary(AM_HAL_WDT_LFRC_CLK_1HZ);
-> 00000011000000000000000000000000
```

Reply:
Ah, important indeed.